src/blueapi/client/client.py

```diff
@@ -46,17 +46,15 @@ def __init__(
     @classmethod
     def from_config(cls, config: ApplicationConfig) -> "BlueapiClient":
         rest = BlueapiRestClient(config.api)
-        if config.stomp is not None:
-            stomp_client = StompClient.for_broker(
-                broker=Broker(
-                    host=config.stomp.host,
-                    port=config.stomp.port,
-                    auth=config.stomp.auth,
-                )
+        assert config.stomp is not None, "By this point, the stomp config should be set"
```

Comment:
This prevents starting a client without a message bus- we can do non-plan running operations without a message bus, is this on purpose?

Reply:
this is just to prevent a type error

Reply:
Blueapi should be able to start up without a message bus, please handle the case where there is no stomp config (don't connect to stomp in this case)

Reply:
ok added that logic